TicketUtil.py

```diff
@@ -140,19 +141,55 @@ def __init__(self, base_url, project_key, ticket_id=None, auth=None, user=None,
         # Call our parent class's init method which creates our requests session.
         super(BugzillaTicket, self).__init__(project_key, ticket_id)
 
+    def craft_ticket_url(self):
+        """
+        Crafts the ticket URL out of the base_url, project_key, and ticket_id.
+        :return: ticket_url: The URL of the ticket.
+        """
+        ticket_url = None
+
+        # If we are receiving a ticket_id, it indicates we'll be doing an update or resolve, so set ticket_url.
+        if self.ticket_id:
+            ticket_url = "{0}/show_bug.cgi?id={1}".format(self.base_url, self.ticket_id)
+
+        return ticket_url
+
 
     def create_ticket_parameters(self, options_dict):
+        """
+        Creates the payload for the POST request when creating a Bugzilla ticket.
+
+        Example:
+        options_dict = {"product" : "TestProduct",
+                        "component" : "TestComponent",
+                        "version" : "unspecified",
+                        "summary" : "'This is a test bug - please disregard",
+                        "alias" : "SomeAlias",
+                        "op_sys" : "All",
+                        "priority" : "P1",
+                        "rep_platform" : "All"}
+        :param options_dict: Key: Value pairs for updating fields in Bugzilla.
+        :return: params: A dictionary to pass in to the POST request containing ticket details.
+        """
+        # Create our parameters for creating the ticket.
+        params = {'fields': {}}
+        params['fields']['project'] = {'key': self.project_key}
 
         params = {"product": str(self.project_key)}
 
         # Iterate through our options and add them to the params dict.
-        for key, value in options_dict.items():
-            params.update({str(key): str(value)})
+        params.update(options_dict)
         return params
 
     def create_requests_session(self):
+        # Returns to the super class if the authentication method is kerberos.
         if self.auth != 'rest':
             return super()
```

Comment:
Corrected and thanks for the information link, I have made the correction and will push the commit later.